src/UsgsAstroLsSensorModel.cpp

```diff
@@ -1698,21 +1663,21 @@ void UsgsAstroLsSensorModel::losToEcf(
    getAdjSensorPosVel(time, adj, xc, yc, zc, vx, vy, vz);
    // CSM image image convention: UL pixel center == (0.5, 0.5)
    // USGS image convention: UL pixel center == (1.0, 1.0)
-   double sampleCSMFull = sample + m_offsetSamples;
+   double sampleCSMFull = sample;
    double sampleUSGSFull = sampleCSMFull;
    double fractionalLine = line - floor(line);
 
    // Compute distorted image coordinates in mm (sample, line on image (pixels) -> focal plane
    std::tuple<double, double> natFocalPlane;
-   natFocalPlane = computeDistortedFocalPlaneCoordinates(fractionalLine, sampleUSGSFull, m_detectorSampleOrigin, m_detectorLineOrigin, m_detectorSampleSumming, m_startingSample, m_detectorLineOffset, m_iTransS, m_iTransL);
+   natFocalPlane = computeDistortedFocalPlaneCoordinates(fractionalLine, sampleUSGSFull, m_detectorSampleOrigin, m_detectorLineOrigin, m_detectorSampleSumming, m_startingSample, 0, m_iTransS, m_iTransL);

```

Comment:
If we're eliminating `m_detectorLineOffset`, can we also just take it out of this function signature and definition?

Reply:
Probably a better question for @jessemapel

Reply:
@kberryUSGS That sounds good